.github/workflows/pkgci_test_onnx.yml

```diff
@@ -40,10 +40,12 @@ jobs:
             numprocesses: 1
             config-file: onnx_ops_gpu_hip_rdna3_O0.json
             runs-on: nodai-amdgpu-w7900-x86-64
-          - name: amdgpu_hip_rdna3_O3
-            numprocesses: 1
-            config-file: onnx_ops_gpu_hip_rdna3_O3.json
-            runs-on: nodai-amdgpu-w7900-x86-64
+          # TODO(#21928): Enable the test suite after we fix the slow tests or
+          # have more machines.
+          # - name: amdgpu_hip_rdna3_O3
+          #   numprocesses: 1
```

Comment:
We could try updating the drivers/firmware on these runners to see if they can handle running HIP programs/tests in parallel now. Running these unit tests serially is quite inefficient.

Reply:
I forgot to ask a qeustion: do we let [onnx_ops_gpu_hip_rdna3_O3.json](https://github.com/iree-org/iree/blob/main/tests/external/iree-test-suites/onnx_ops/onnx_ops_gpu_hip_rdna3_O3.json) be unused? I can't get an action for the file, so I just ignore it now.

Reply:
It's fine to keep it there and not have a CI job using it.

Another thing you could do would be to create a json file for a smaller number of tests that have had issues before. I think the ONNX op tests are opt-out while the ONNX model tests are opt-in, so you would unfortunately need to move all tests you want to skip to `skip_compile_tests`... or change the test suite (see also https://github.com/iree-org/iree-test-suites/issues/25)

Reply:
I'll leave it unused. @kuhar is going to take a look tomorrow, so it is fine. We can follow up on this if it becomes tricker.